src/app/calendar-app/runbox-calendar-event.ts

```diff
@@ -273,4 +577,14 @@ export class RunboxCalendarEvent implements CalendarEvent {
             return moment.utc(utc.toString()).tz(moment.tz.guess());
         }
     }
+
+    private momentToIcalTime(input: moment.Moment, zone: ICAL.Timezone): ICAL.Time {
+        if (!zone) {
+            // This should probably use tz.guess() ?
```

Comment:
Probably, like the method above.

Reply:
This shaves a whole new yak, as if we guess and find user's timezone (or grab the one configured in the account), we now need to have a VTIMEZONE entry to store with it. I'm now looking into https://github.com/libical/vzic as a source of these.

Reply:
Sorted - that was fun..